pkg/server/query/server.go

```diff
@@ -293,7 +292,7 @@ func renderTemplate(ctx *fasthttp.RequestCtx, tpl *template.Template) {
 func (hs *queryserverCfg) RunSafeServer() error {
 	hs.Router.GET("/health", hs.Recovery(getSafeHealthHandler()))
 	var err error
-	hs.ln, err = reuseport.Listen("tcp4", hs.Addr)
+	hs.ln, err = net.Listen("tcp4", hs.Addr)
```

Comment:
This server should also use `tcplisten` instead of `net`, right? Same comment for line 234